### ISSUE TITLE
Unable to set up proxy

### ISSUE THREAD

**Issue by asteriosgr** [grafana/xk6-browser#449]
As the title suggests I am unable to initiate the xk6-browser using a proxy. I am aware that the README is a bit wrong on the way it defines the proxy, because proxy support through variables is still on TODO. 

https://github.com/grafana/xk6-browser/blob/main/common/connection.go#L146

So currently the proxy value is in theory taken from the environment variables.

However when exporting  them, a proxy is still not used. For example:

export http_proxy='http://proxy.net:43'
export https_proxy='http://proxy.net:43'
export HTTP_PROXY='http://proxy:43'
export HTTPS_PROXY='http://proxy:43'

Is there any workaround for this issue?

**Comment by inancgumus** [comment]
Hi @asteriosgr,

[This](https://github.com/grafana/xk6-browser/blob/main/common/connection.go#L146) is for connecting to the Browser's WebSocket DevTools URL.

To fix this issue, I need some information from you. Could you tell me your motivation for using a proxy in this case? Is it about passing proxy information to the launched browser and letting the browser use the proxy?

**Comment by asteriosgr** [comment]
Hi @inancgumus . I am trying to test a website using playwright. The website is accessible only through a specific proxy. With plain playwright, I am able to set up a proxy in the following way:
`proxy: { server: PROXY},`

However, with the xk6 browser, I tried setting proxy also as a chrome argument (flag), and also through the way it is mentioned in the README https://github.com/grafana/xk6-browser#launch-options . In both cases I was unable to establish connection to the PROXY. Then going through the code I noticed that you should try exporting it as an env variable, https://pkg.go.dev/net/http#ProxyFromEnvironment . Once again this did not do the trick.

So, TLDR, I simply need to access a website through a PROXY. However, I am unable to set up a proxy.

**Comment by inancgumus** [comment]
@asteriosgr, I can let you pass the `proxy` options to the browser if that works for you (since we're also missing and not passing these options to the browser yet). Could you check whether you can access the website by launching chrome with these command-line arguments:

```
<path-to-your-chrome-executable> --proxy-server="<your-proxy-url-here>"
```

For example (chromium is my `$PATH` environment variable):

```
chromium --proxy-server="https://myproxy:myproxyport"
```

**Comment by asteriosgr** [comment]
Well as I mentioned earlier, I am starting chrome with the following args:
const browser = launcher.launch(‘chromium’, { headless: false, args: [`--proxy-server=“http://myproxy:port”`,`--no-sandbox`, ‘--ignore-certificate-errors’, ‘--incognito’], proxy: { server: myproxy:myport}});

Where myproxy and myport of course is the proxy value and the port number I am using.

**Comment by inancgumus** [comment]
@asteriosgr, what I mean by my previous message was that **instead of** running via xk6-browser or Playwright, **starting Chrome directly from the command line with the proxy arguments**, going to the website behind a proxy, and telling me if it works that way. We'll try to fix the `args` if it works.

**Comment by asteriosgr** [comment]
On my end, setting the `--proxy-server` argument, is not doing anything in the local chrome browser.
